tensorflow/core/kernels/data/parallel_map_dataset_op.h

```diff
@@ -0,0 +1,58 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
```

Comment:
please merge the logic in `parallel_map_iterator.h` header file into this one and get rid of the `parallel_map_iterator.h`